client/src/assets/clippy.svg

```diff
@@ -1 +1,11 @@
-<svg viewBox="0 0 1000 1000" height="30" width="30" xmlns="http://www.w3.org/2000/svg"><path d="M704 896H64V320h640v192h64V192c0-35-29-64-64-64H512C512 57 455 0 384 0S256 57 256 128H64c-35 0-64 29-64 64v704c0 35 29 64 64 64h640c35 0 64-29 64-64V768h-64v128zM192 192h64s64-29 64-64 29-64 64-64 64 29 64 64 32 64 64 64h64s64 29 64 64H128c0-39 28-64 64-64zm-64 512h128v-64H128v64zm448-128V448L320 640l256 192V704h320V576H576zM128 832h192v-64H128v64zm320-448H128v64h320v-64zM256 512H128v64h128v-64z"/></svg>
\ No newline at end of file
+<svg
+  viewBox="0 0 1000 1000"
+  height="30"
+  width="30"
+  xmlns="http://www.w3.org/2000/svg"
```

Comment:
You might want to add `role="img"` and/or link the `<title>` with `aria-labelledby`.

Reply:
I have added the `role="img"` attributes.
Is `aria-labelledby` needed even with the `<title>` tag?

Reply:
I'll give it a try and let you know.

Reply:
Okay, the `<title>` was read everywhere except the badges (which is to be expected, given that it is a `<symbol>` as parent).

No need for aria-labelledby!

Reply:
Chrome does recognize `<title>` in `<symbol>`.

![image](https://user-images.githubusercontent.com/25801283/122993857-6d421f00-d3c5-11eb-8e78-99a5e1440d66.png)